### DISCUSSION TITLE
problems installing Brew on Catalina 10.15.7

### DISCUSSION THREAD

**Discussion by shorty310** [Homebrew/discussions#586] [Getting started] [answered]
My default shell on Catalina is zsh. I have a terminal window open. I am unable to run the bash shell to install Brew. I thought that the "/bin/bash" at the beginning of the command would override the default shell and this should work. Any tips on how to install Brew on Catalina would be appreciated. I am trying to install Brew so that I can use it to install the newer version of Python.

```
>>> /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
  File "<stdin>", line 1
    /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
    ^
SyntaxError: invalid syntax
```

**Comment by carlocab** [comment, marked as answer]
Are you sure you're not actually running Python? Because that's exactly what it looks like if I run 

    /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"

in Python.

**Reply by shorty310** [reply]
Thanks a lot. You were correct.
 
I was able to install Homebrew after I exited Python.
